howto/README.md

```diff
@@ -0,0 +1,49 @@
+# How do I ...
+
+## ... start a Rails app?
+
+Use [suspenders]:
+
+```ruby
+$ gem install suspenders
+$ suspenders the-name-of-your-project-here
+$ cd the-name-of-your-project-here/
+$ rake
+```
+
+[suspenders]: https://github.com/thoughtbot/suspenders
+
+## ... feature-test a Rails app's Javascript?
+
+Use [capybara-webkit]. In your `Gemfile`:
+
+```ruby
+gem "capybara-webkit"
+```
+
+In `spec/support/capybara_webkit.rb` (for Rspec):
+
+```ruby
+Capybara.javascript_driver = :webkit
+
+Capybara::Webkit.configure do |config|
+  config.block_unknown_urls
+end
+```
+
+When writing a spec, to make use of this you must set the `:js` flag for that
```

Comment:
How about: `When writing a spec, you must set the :js flag to make use of capybara-webkit.`?